### ISSUE TITLE
script to populate correspContext Element

### ISSUE THREAD

**Issue by csae8092** [emt-project/emt-working-data#12]
see e.g. https://schnitzler-briefe.acdh.oeaw.ac.at/L00074.xml

```xml
<correspContext>
    <ref type="belongsToCorrespondence" target="correspondence_10815">Bahr, Hermann</ref>
    <ref subtype="previous_letter" type="withinCollection" target="L00073">Bahr an Schnitzler, [20. 2. 1892]</ref>
    <ref subtype="next_letter" type="withinCollection" target="L00075">Hofmannsthal an Schnitzler, 23. 2. 1892</ref>
    <ref subtype="previous_letter" type="withinCorrespondence" source="correspondence_10815" target="L00073">Bahr an Schnitzler, [20. 2. 1892]</ref>
    <ref subtype="next_letter" type="withinCorrespondence" source="correspondence_10815" target="L00081">Bahr an Schnitzler, [13. 3. 1892]</ref>
</correspContext>
```

`<ref type="belongsToCorrespondence" target="correspondence_10815">` could be derived from `<correspAction type="sent">` respectively from `<correspAction type="received">`

* for next/previous elements the letters needs to be sorted which means we can only process letters which are properly sorted

- [ ] create a script which reports the number of letters with a full date; how many letters were written on the same date; how many letters can not be dated

basically something similar as https://github.com/emt-project/emt-static/blob/main/make_calendar_data.py

**Comment by ttechnicus** [comment]
@SpitzbartAnna Please discuss with Frau Keller the options of navigation:
- by correspondence ("thread") - this is what I'm developing right now
- by archival order (or please name this option somehow) - this is presently on the website
- chronologically - this would seem logical, shall it replace the archival order, or be a 3rd option?
